Networking/Networking/Model/Site.swift

```diff
@@ -37,6 +37,10 @@ public struct Site: Decodable {
     ///
     public let timezone: String
 
+    /// Return the website UTC time offset, showing the difference in hours and minutes from UTC, from the westernmost (−12:00) to the easternmost (+14:00).
+    ///
+    public let gmtOffset: Decimal
```

Comment:
Curious to know why a Decimal instead of a Double? (Not complaining!)

Reply:
(I had the same question just out of curiosity!)

Reply:
Mainly because of rounding. It costs a little more memory but doesn't have rounding troubles like double sometimes has. But it this case I can use without doubt also Double, so I'm going to change it.

Reply:
I don't think it needs changed. 🙂

I didn't know double had problems with rounding, but that makes sense. I know double has problems with precision when used in math and calculating currency values.